### PR TITLE
Feat/ot307 21 generate migration controller testimonial

### PR DESCRIPTION
**Pull request by AlexMamani85**
# Ticket description: Crear modelo, migración y controlador base de Testimonials

Ticket_number#21 (issue)

# Checklist:

- [ok ] generate model Testimonial
- [ok ] migration Testimonial
- [ok ] add discard to Testimonials
- [ok ] migration discard to Testimonials
- [ok ] rails active_storage:install

### CODE REVIEW

File: db/migrate/20220916181516_create_active_storage_tables.active_storage.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  private
+    def primary_and_foreign_key_types
+      config = Rails.configuration.generators
+      setting = config.options[config.orm][:primary_key_type]
+      primary_key_type = setting || :primary_key
+      foreign_key_type = setting || :bigint
+      [primary_key_type, foreign_key_type]
+    end
```
Comment (fciballero) [inline db/migrate/20220916181516_create_active_storage_tables.active_storage.rb:47]: Did you use the command `rails active_storage:install` to generate this migration file?

File: app/models/testimonial.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+# Indexes
+#
+#  index_testimonials_on_discarded_at  (discarded_at)
+#
+class Testimonial < ApplicationRecord
+  include Discard::Model
+  has_one_attached :testimonial_image
```
Comment (fciballero) [inline app/models/testimonial.rb]: In this case it is not necessary to use `testimonial_image`:
```suggestion
  has_one_attached :image
```

**Review by fciballero — COMMENTED** [review]
Good Job Alex!!! , I've left a comment.

File: db/migrate/20220916181516_create_active_storage_tables.active_storage.rb (hunk 1 above)
Comment (AlexMamani85) [reply db/migrate/20220916181516_create_active_storage_tables.active_storage.rb:47]: yes, indeed

**Pull request closed without merging**